src/styles/mixins.scss

```diff
@@ -74,8 +74,14 @@ $spacer: 1.875rem; // 30px
 @mixin selects(
 	$primary-background-color: $default-primary-background-color,
 	$closed-background-color: $default-primary-background-color,
+
 	$selected-color: $default-selected-color,
 	$selected-background-color: $default-selected-background-color,
+
+	// By default, we style selected & hover the same, but allow consuming projects to pass in separate values if desired
```

Comment:
Eek, at some point we should change this as this is not a great default 😬

Reply:
Oh, we can just change that now if you want; just a different shade of gray for `hover`, perhaps?

Reply:
Fixed: e41f88d9a6c46955f0d05eeada463f455eccf905

![Screen Shot 2020-09-29 at 1 06 00 PM](https://user-images.githubusercontent.com/14101491/94590420-a42f3e00-0254-11eb-9c29-7eaf0d6daa27.png)